lab_1_classify_by_unigrams/main.py

```diff
@@ -4,51 +4,123 @@
 """
 
 
-def tokenize(text: str) -> list[str] | None:
-    """
-    Splits a text into tokens, converts the tokens into lowercase,
-    removes punctuation, digits and other symbols
-    :param text: a text
-    :return: a list of lower-cased tokens without punctuation
+def tokenize(text: str) -> list[str]:
     """
+        Splits a text into tokens, converts the tokens into lowercase,
+        removes punctuation, digits and other symbols
+        :param text: a text
+        :return: a list of lower-cased tokens without punctuation
+        """
+    if type(text) != str:
```

Comment:
Consider using `isinstance`: type equality check is a bad practice.

Reply:
Here and everywhere else

Reply:
Done